lib/parser.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+const path = require('path')
+
+module.exports = class Parser {
+
+  static commandParser (opts) {
+    const command = opts.command
+    if (!command) return null
+    if (Array.isArray(command)) {
+      return command.join(' ')
+    }
+    return command
+  }
+
+  static buildDockerfilePathParser (opts) {
+    const build = opts.build
+    const warnings = opts.warnings
+    let dockerFileName = 'Dockerfile'
+    if (!build) return null
+
+    if (build.args) {
+      // TODO: Maybe pass these to the ENVs?
+      warnings.push({
```

Comment:
I didn't add logic around it being `undefined` because we're the ones who pass it in. This is not a user input. Do you think we should add them?